doc/man3/SSL_SESSION_get_time.pod

```diff
@@ -66,7 +66,9 @@ of the session.
 SSL_SESSION_get_time_ex() and SSL_SESSION_get_timeout() return the currently
 valid values.
 
-SSL_SESSION_set_time_ex() and SSL_SESSION_set_timeout() return 1 on success.
+SSL_SESSION_set_time_ex() returns time on success.
+
+SSL_SESSION_set_timeout() returns 1 on success.
 
```

Comment:
While about it could we also document the return values for the other functions in the synopsis not mentioned here? e.g. SSL_get_timeout/SSL_set_timeout/SSL_SESSION_get_time/SSL_SESSION_set_time/SSL_get_time/SSL_set_time